config/boards/aml-s805-mxq.tvb

```diff
@@ -0,0 +1,19 @@
+# MXQ TV box based on Amlogic S805 quad core 1GB RAM SoC GBE
+BOARD_NAME="MXQ"
+BOARDFAMILY="meson8b"
+BOARD_MAINTAINER=""
+KERNEL_TARGET="current,edge"
+KERNEL_TEST_TARGET="current"
+BOOTCONFIG="none"
+BOOTSCRIPT="boot-aml-s805-mxq.cmd:boot.cmd"
+# The preinstalled U-BOOT looks for this script
+BOOTSCRIPT_OUTPUT="s805_autoscript"
+BOOTENV_FILE="aml-s805-mxq.txt"
+
+BOOTSIZE="200"
+BOOTFS_TYPE="fat"
+BOOT_LOGO=desktop
+
+family_tweaks() {
```

Comment:
Rename to:
function post_family_tweaks__aml_s805_logo() {

And does logo shows?

Reply:
I don't recall seeing the logo. I don't install the image to builtin memory. I rely on the factory intalled u-boot to load armbian from SD card. That's why some content of the boot partition are unused, at least for now. This fragment is left over after starting from other board's config.
I will remove it.

I am away from home and have no access to the test device. I will update and test, but it will be sometime in January.

Reply:
> I will remove it.

OK, it probably doesn't work.

> but it will be sometime in January.

No worries.